docs/operator-docs/naming-executions.md

```diff
@@ -0,0 +1,78 @@
+---
+id: naming-executions
+title: Naming Executinons
+hide_title: true
```

Comment:
Can we set this to `false` and then remove the manually added top level heading? It's a minor formatting benefit, but I just changed all the docs to be consistent about this

Reply:
3dbac0720